package.json

```diff
@@ -194,6 +194,9 @@
     "rollup-plugin-typescript":
       "0.8",
 
+    "rollup-plugin-pug":
+      "0.1.2",
```

Comment:
Do not fixate `patch`. This isn't necessary. Just stick to `0.1`, that's enough.